rs/orchestrator/src/boundary_node.rs

```diff
@@ -164,7 +170,15 @@ impl BoundaryNodeManager {
             OrchestratorError::IoError("unable to read ic-boundary environment variables".into(), e)
         })?;
 
-        let args = vec![format!("--tls-hostname={}", domain_name)];
+        let args = vec![
+            format!("--tls-hostname={}", domain_name),
+            format!("--obs-log-anonymization-canister-id={LOG_ANONYMIZATION_CID}"),
```

Comment:
Should this instead be part of the new [ic-boundary.env](https://sourcegraph.com/github.com/dfinity/ic/-/blob/ic-os/components/ic/share/ic-boundary.env)?

Reply:
Yes @rikonor I think it should go there if it's not dynamically created by the orchestrator

Reply:
Moved to the env file 👍 thanks for the tip 🙂